src/index.js

```diff
@@ -12,7 +12,7 @@ server.listen(port, error => {
     console.log(error);
   }
 
-  console.log(`Started and listening on http://localhost:${port}`);
+  console.log(`Started and listening on http://localhost:${port}/article/article-id`);
```

Comment:
It isn't listening on the route, it's listening on the port and we happen to only have one route defined. I don't think this should be changed at all.